index.html

```diff
@@ -171,45 +205,49 @@ <h5 class="btn-connect">LET’S CONNECT</h5>
-    <section class="footer-section"> 
-    <footer class="footer-form">
-        <div class="footer-bg">
-        <div class="form-heading">
-            <h2 class="contact-heading">Contact me </h2>
-            <p class="contact-detail">If you have an application you are
-                 interested in developing, a feature that 
-                  you need built or a project that needs
-                   coding. I’d love to help with it
-                 </p>
+            <div class="framworks">
+                Framworks <span>></span>
+            </div>
+            <div class="line"></div>
+            <div class="framworks">
+                Skills <span>></span>
+            </div>
+            <div class="line"></div>
         </div>
-
-        <form action="https://formspree.io/f/xlevdjbz" method="post">
-            <ul>
-                <li>
-                    <input class="name" type="text" maxlength="30" name="name"  placeholder="Full Name" required>
-                </li>
-                <li>
-                    <input class="email" type="email" name="email" placeholder="Email Address" required>
-                </li>
-                <li>
-                    <textarea class="textarea" maxlength="500" placeholder="write yout message here" required name="message" cols="30" rows="5"></textarea>
-                </li>
-            </ul>
-            <button class="btn-touch" type="submit">Get in Touch</button>
-        </form>
     </div>
-    </footer>
-</section>
+    </section>
+    <!-- footer section start -->
+    <section class="footer-section">
+        <footer class="footer-form">
+            <div class="footer-bg">
+                <div class="form-heading">
+                    <h2 class="contact-heading">Contact me </h2>
+                    <p class="contact-detail">If you have an application you are
+                        interested in developing, a feature that
+                        you need built or a project that needs
+                        coding. I’d love to help with it
+                    </p>
+                </div>
+
+                <form action="https://formspree.io/f/xlevdjbz" method="post">
+                    <ul>
+                        <li>
+                            <input class="name" type="text" maxlength="30" name="name" placeholder="Full Name" required>
+                        </li>
+                        <li>
+                            <input class="email" type="email" name="email" placeholder="Email Address" required>
+                        </li>
+                        <li>
+                            <textarea class="textarea" maxlength="500" placeholder="write yout message here" required
+                                name="message" cols="30" rows="5"></textarea>
+                        </li>
+                    </ul>
+                    <button class="btn-touch" type="submit">Get in Touch</button>
+                </form>
+            </div>
+        </footer>
```

Comment:
- [ ] Great job but I think here the width is not adjusted as the Figma so kindly fix the width for all heading and paragraph and the form inputs to match the figma design to make your website looks nicer ✅🔥💯💯

- your design
![image](https://user-images.githubusercontent.com/58553711/187443203-8f7e70a4-7ace-48e5-a819-7d0ade9abe33.png)

- Figma design
![image](https://user-images.githubusercontent.com/58553711/187443287-7a927916-04dd-44c4-94cc-e3ad819139bd.png)